docusaurus/src/pages/markdown-page.md

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
Commit message nit: Also capitalize "Markdown" in the title, like you did in the body.